src/components/comment/comment.test.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import type { PleasantestUtils } from 'pleasantest';
```

Comment:
TIL `import type` 👍

Reply:
It is relatively new! It makes things easier for tools that are not type-aware (like babel) to know which imports need to be kept in the output, vs which imports are only used for types so they can't be in the output